index.js

```diff
@@ -0,0 +1,123 @@
+const http = require('node:http');
+const fs = require('node:fs');
```

Comment:
This can be removed since its not used anywhere :)

Reply:
I guess I can remove fs only, as I use http module to create sserver.

Reply:
Fixed